bundle/Form/Field/NgHtmlTextValueTransformer.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\IbexaFieldTypeHtmlTextBundle\Form\Field;
+
+use Ibexa\AdminUi\Form\DataTransformer\FieldType\FieldValueTransformer;
+
+class NgHtmlTextValueTransformer extends FieldValueTransformer
```

Comment:
Why do we need an empty transformer? For validation you mentioned that is missing or?

Reply:
As we discussed in person, this file will be edited in the next commit (when adding validation) so we'll keep it here for now.